dmutils/content_loader.py

```diff
@@ -413,6 +404,30 @@ def _get_single_question_data(self, form_data):
 
         return {self.id: value if value else None}
 
+    def get_error_messages(self, errors):
+        error_fields = set(errors.keys()) & set(self.form_fields)
+        if not error_fields:
+            return {}
+
+        question_errors = {}
```

Comment:
Would this not mean that the order of errors within a multi-question would be undefined?